pkg/imageset/updateinstaller.go

```diff
@@ -271,7 +271,7 @@ func getClient(kubeConfig *rest.Config) (client.Client, error) {
 		MapperProvider: apiutil.NewDiscoveryRESTMapper,
 	}
 
-	mapper, err := managerOptions.MapperProvider(kubeConfig)
+	mapper, err := managerOptions.MapperProvider(kubeConfig, managerOptions.Client.HTTPClient)
```

Comment:
`managerOptions.Client.HTTPClient` will be `nil`, right? And that'll [blow up](https://github.com/kubernetes-sigs/controller-runtime/blob/30eae58f1b984c1b8139dd9b9f68dd2d530ed429/pkg/client/apiutil/restmapper.go#L36), right?